tests/routes/test_documents_deprecated.py

```diff
@@ -0,0 +1,633 @@
+        )
+        assert response.status_code == 201
+
+    # Test properties
+    get_detail_response_2 = client.get(
+        f"/api/v1/documents/{response2_document['import_id']}",
+    )
+    assert get_detail_response_2.status_code == 200
+
+    # Check some expected properties of the returned document
+    get_detail_json_2 = get_detail_response_2.json()
+    assert (
+        get_detail_json_2["name"]
+        == "Agriculture Sector Strategy 1487-1491 (2008/9-2013/4)"
+    )
+    assert get_detail_json_2["description"] == "the document description"
+    assert get_detail_json_2["postfix"] == "postfix2"
+    assert get_detail_json_2["publication_ts"] == "1999-01-01T00:00:00"
+    assert (
+        get_detail_json_2["source_url"]
+        == "https://climate-laws.org/rails/active_storage/blobs/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcG9IIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--be6991246abda10bef5edc0a4d196b73ce1b1a26/g"
+    )
+
+    assert get_detail_json_2["source"] == {"name": "may it be with you"}
+    assert get_detail_json_2["geography"] == {
+        "display_value": "not my favourite subject",
+        "slug": "not-my-favourite-subject",
+        "value": "NMFS",
+        "type": "country",
+    }
+    assert get_detail_json_2["type"] == {"name": "just my type", "description": "sigh"}
+    assert get_detail_json_2["languages"] == [
+        {
+            "language_code": "afr",
+            "part1_code": None,
+            "part2_code": None,
+            "name": "Afrikaans",
+        }
+    ]
+    assert get_detail_json_2["category"] == {
+        "name": "a category",
+        "description": "a category description",
+    }
+
+    sorted_related_docs = sorted(
+        get_detail_json_2["related_documents"],
+        key=lambda d: d["document_id"],
+    )
+    assert sorted_related_docs == [
+        {
+            "document_id": response1_document["id"],
+            "import_id": "CCLW.001.000.XXX",
+            "name": "Energy Sector Strategy 1387-1391 (2007/8-2012/3)",
+            "postfix": "postfix1",
+            "description": "the document description",
+            "country_code": "NMFS",
+            "country_name": "not my favourite subject",
+            "publication_ts": "2000-01-01T00:00:00",
+            "slug": "not-my-favourite-subject_2000_energy-sector-strategy-1387-1391-2007-8-2012-3_000_xxx",
+        },
+        {
+            "document_id": response3_document["id"],
+            "import_id": "CCLW.003.000.XXX",
+            "name": "Energy Sector Strategy 1387-1391 (2009/8-2014/3)",
+            "postfix": "",
+            "description": "the document description",
+            "country_code": "NMFSA",
+            "country_name": "not my fav subject again",
+            "publication_ts": "1998-01-01T00:00:00",
+            "slug": "not-my-fav-subject-again_1998_energy-sector-strategy-1387-1391-2009-8-2014-3_000_xxx",
+        },
+    ]
+
+    assert get_detail_json_2["events"] == document2_payload["events"]
```

Comment:
This is arguably slightly hard to read, could the get_detail_json_2 just be compared against another json object?